Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
```

Comment:
When I wrote this, I intentionally used the "bare" ubuntu image in order to expose missing package dependencies in CI.